### DISCUSSION TITLE
能单线程一个视频一个视频去看吗

### DISCUSSION THREAD

**Discussion by 2682856025** [liuyunfz/chaoxing_tool#63] [Q&A]
多个视频同时看会被清进度，怎么进行答题呀

**Comment by SongZihui-sudo** [comment]
liuyunfz/chaoxing_tool at dev (github.com)&nbsp;可以看下 dev 分支




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "liuyunfz/chaoxing_tool"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月13日(星期四) 晚上8:58
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[liuyunfz/chaoxing_tool] 能单线程一个视频一个视频去看吗 (Discussion #63)





 
多个视频同时看会被清进度，怎么进行答题呀
